docs/user/ovirt/install_ipi.md

```diff
@@ -0,0 +1,65 @@
+# Install using oVirt platform provider
+
+## Overview
+This provider enables the Openshift Installer to provision VM resources in an \
```

Comment:
This is documentation only meant for human eyes so the `\` are redundant. It's ok to break a sentence on multiple lines. :)